python/plugins/processing/algs/otb/description/5.6.0/ComputeOGRLayersFeaturesStatistics.xml

```diff
@@ -5,26 +5,26 @@
   <group>Segmentation</group>
   <description>Compute statistics of the features in a set of OGR Layers</description>
   <parameter>
-    <parameter_type source_parameter_type="ParameterType_InputVectorData">ParameterVector</parameter_type>
+    <parameter_type source_parameter_type="ParameterType_InputFilename">ParameterFile</parameter_type>
     <key>inshp</key>
     <name>Name of the input shapefile</name>
     <description>Name of the input shapefile</description>
-    <shapetype />
+    <isFolder />
     <optional>False</optional>
   </parameter>
   <parameter>
     <parameter_type source_parameter_type="ParameterType_OutputFilename">OutputFile</parameter_type>
     <key>outstats</key>
-    <name>XML file containing mean and variance of each feature.</name>
-    <description>XML file containing mean and variance of each feature.</description>
+    <name>Output filename XML file containing mean and variance of each feature.</name>
+    <description>XML file containing mean and variance of each feature(.csv format)</description>
+    <optional>.xml</optional>
   </parameter>
   <parameter>
     <parameter_type source_parameter_type="ParameterType_StringList">ParameterString</parameter_type>
     <key>feat</key>
-    <name>List of features to consider for statistics.</name>
+    <name>List of feield names in Input seg polygon to consider for statistics e.g. "keyword" "keyword2"</name>
```

Comment:
I changed to segmentation and modified mistyping about field. Thank you. I used short name : seg considering the space in the window.